.github/workflows/build-and-test-dotnet.yml

```diff
@@ -153,7 +153,7 @@ on:
       timeout-minutes:
         type: number
         default: 360
-        description: Configuration for the timeout-minutes parameter of the workflow. The 360 is GitHub's default.
+        description: Configuration for the timeout-minutes parameter of the workflow. 360 is GitHub's default.
```

Comment:
The "the" in these referred to the default, but it's bad English, and in some occasions, the default wasn't even 360.